languages/ruby/exercises/concept/conditionals-ternary/.docs/introduction.md

```diff
@@ -18,4 +18,4 @@ can be re-written as:
 traffic_light == 'green' ? cross_the_road : wait
 ```
 
-The code on the left side of the `?` is the condition and the code on the right contains the two possible branches, separated by the `:`. If the condition is *true*, the code on the *left* side of the `:` is executed; if the condition is *false*, then the code on the *right* of the `:` gets executed.
+The code on the left side of the `?` is the condition and the code on the right contains the two possible branches, separated by the `:`. If the condition is _true_, the code on the _left_ side of the `:` is executed; if the condition is _false_, then the code on the _right_ of the `:` gets executed.
```

Comment:
We could call "left side of the :" as "the true branch" as well.

Reply:
> We could call "left side of the :" as "the true branch" as well.

Someone might not know which side was true and which was false without being told here, so I think it should be left+right.

Reply:
Yes, not without introduction of the true branch coming first.  I mean thereafter.   This is what we are doing here, the introduction to how it will be referenced.  I am saying, that we must use left and right, of course, as that is definitive, but we might reference the content later as the "true branch" of the ternary, rather than the longer "left side of the :" reference.  The set up is happening here in either case.

Reply:
"here" being in this document, not necessarily this line.

Reply:
Yep. Gotcha. I like that.